_tests/integration/fastlane_test.go

```diff
@@ -128,6 +128,7 @@ configs:
               inputs:
               - lane: $FASTLANE_LANE
               - work_dir: $FASTLANE_WORK_DIR
+              - enable_cache: "no"
```

Comment:
Could the iOS + Android cache steps be added here?

Reply:
After a deeper look, I don't think it's a good idea. This workflow is offered when the user selects a raw Fastlane workflow (even though the scanner recognizes the project as iOS/Android too). So I don't see how we could select here one of the Gradle/SPM/Carthage/Cocoapods cache steps.

I also wanted to include an empty generic cache step pair, but then realized that they have required inputs, I can't leave the cache key unfilled.